csm/conf/csm_default.conf

```diff
@@ -88,6 +88,7 @@ CSM:
   username: csm
   password: ''
   password_decryption_key: cortx
+  session_backend: in_memory
```

Comment:
There should be two possible values for session_backend
1. persistent  -> for consul
2. local -> for in_memory

Reply:
@rohitkolapkar I feel persistent is quite generalize keyword, 
As @pawankumarsrivastava  mentioned in the discussion that we can store in File also or they are might be something new.